docs/docs/how-to/sourcing-data/sourcing-from-kontent-ai.md

```diff
@@ -0,0 +1,231 @@
+
+You can also take advantage of Kontent.ai ability to deliver content in multiple languages and create relationships among various content with linked items. However you structure your content in Kontent.ai, the [source plugin](https://github.com/kontent-ai/gatsby-packages/tree/master/packages/gatsby-source) will ensure the proper nodes are created for your Gatsby site.
+
+\_Note: For this guide, you'll work from scratch from the [Gatsby default starter](https://github.com/gatsbyjs/gatsby-starter-default) to get a feel for how sourcing from Kontent.ai works. If you'd like to look at an example of a complete basic site, have a look at the [Kontent.ai Gatsby starter site](https://github.com/kontent-ai/gatsby-starter), which includes worked examples for querying content.
+
+## Setup
+
+### Kontent.ai
+
+The first thing to do, if you haven't already done so, is [sign up for a Kontent.ai account](https://app.kontent.ai/sign-up?utm_medium=gatsbyjsorg&utm_source=guide). This will automatically start a free 30-day trial with all Kontent.ai features. At any point during the trial or after, you can switch to a Developer plan (which always starts as free) or a higher plan with more features.
+
+Once you have a subscription to Kontent.ai, you need some content to retrieve. If you know what you want, you can set up your own content types (templates for your content) and then create content items (the actual content) based on them. If you'd like to take a shortcut and see some example content, you can use [Sample Project generator](https://app.kontent.ai/sample-project-generator) to generate project "Sample Project" and import sample content. This guide will continue with the example of the "Sample Project".
+
+The created Sample Project is a comprehensive presentation of a fictional coffee company named Dancing Goat that showcases various Kontent.ai features. It can be displayed in various channels, as you can see by going to the [Quickstart page in Kontent.ai](https://app.kontent.ai/quickstart) from within that project.
+
+For this guide, you don't have to worry about most of the features. You'll pull some data to display within the Gatsby site you'll create in the next step. The only thing you need to get is your Project ID, which you can find in Kontent.ai under _Project settings_ -> _API keys_.
+
+### Gatsby
+
+#### Adding content to existing pages
+
+Now that you have some content to pull, you can create a basic Gatsby site to display the content. Assuming you have the [Gatsby CLI installed](/docs/quick-start/#install-the-gatsby-cli), create a new site and navigate to it in your terminal:
+
+```shell
+gatsby new kontent-guide
+cd kontent-guide
+```
+
+Next, install the [Kontent.ai source plugin](https://github.com/kontent-ai/gatsby-packages/tree/master/packages/gatsby-source):
+
+```shell
+npm install --save @kontent-ai/gatsby-source
+```
+
+Once that's done, you need to add the plugin to `gatsby-config.js`:
+
+```javascript
```

Comment:
```suggestion
```js:title=gatsby-config.js
```

Reply:
What is this doing?

Reply:
https://www.gatsbyjs.com/contributing/gatsby-style-guide/#code-formatting-titles